numba/core/pylowering.py

```diff
@@ -170,6 +167,15 @@ def lower_inst(self, inst):
         else:
             raise NotImplementedError(type(inst), inst)
 
+    @utils.cached_property
+    def _omitted_typobj(self):
+        """Return a `OmittedArg` type instance in LLVM value suitable for
```

Comment:
```suggestion
        """Return a `OmittedArg` type instance as a LLVM value suitable for
```

Reply:
updated in https://github.com/numba/numba/pull/5922/commits/ebbae83720e6c29ef6fbb756298211d24869279f